packages/app-accounts/src/modals/ChangePass.tsx

```diff
@@ -81,22 +80,31 @@ class ChangePass extends React.PureComponent<Props, State> {
           {t('Change account password')}
         </Modal.Header>
         <Modal.Content>
-          <AddressSummary value={account.address()} />
-          <Password
-            autoFocus
-            isError={!isOldValid}
-            label={t('your current password')}
-            onChange={this.onChangeOld}
-            tabIndex={1}
-            value={oldPass}
-          />
-          <Password
-            isError={!isNewValid}
-            label={t('your new password')}
-            onChange={this.onChangeNew}
-            tabIndex={2}
-            value={newPass}
-          />
+          <AddressRow
+            isInline
+            value={account.address()}
+          >
+            <p>{t('Change your password for the account. This will apply to any future use of this account as stored on this machine. Ensure that you securely store this new password and that is is strong and unique to the account.')}</p>
```

Comment:
Typo. As I used the modal, it felt duplicate to say "change password" again as it's already in the title.
Suggest using "browser". Should we add some kind of warning that any Backup file downloaded before will not use this password. a new backup file should probably be downloaded?

```suggestion
            <p>{t('This will apply to any future use of this account as stored on this browser. Ensure that you securely store this new password and that it is strong and unique to the account.')}</p>
```